src/tls.rs

```diff
@@ -590,7 +590,7 @@ extern fn send_alert(ssl: *mut SSL, level: crypto::Level, alert: u8) -> c_int {
     );
 
     let error: u16 = TLS_ALERT_ERROR + u16::from(alert);
-    conn.error = Some(error);
+    conn.error = Some(u64::from(error));
```

Comment:
Probably better to just change `error` on the line above to be u64, so we don't go u8 -> u16 -> u64 (probably needs changing `TLS_ALERT_ERROR` type as well).